generator/generator.py

```diff
@@ -1,31 +1,67 @@
-#import numpy
+import numpy as np
 import sys
 import csv
 import random
+import matplotlib.pyplot as plt
+import scipy.stats
 
 def generator(shards, deltaT, N, mean, std):
```

Comment:
also what is `N`?

Reply:
size of vector

Reply:
so maybe we should name it appropriately?

Reply:
now it is